index.js

```diff
@@ -116,6 +115,14 @@ AFRAME.registerComponent('controller-cursor', {
   },
 
   /**
+   * Remove mesh.
+   */
+  remove: function () {
+    var cursorEl = this.el;
+    cursorEl.removeObject3D('cursormesh');
```

Comment:
Does `remove` detach event listeners? I think you might have to call `pause` manually.